lib/reverb/Reverb.cc

```diff
@@ -458,3 +458,9 @@ void MixxxPlateX2::processBuffer(const sample_t* in, sample_t* out, const uint f
 	}
 }
 
+void MixxxPlateX2::setSampleRate(const double sampleRate) {
+	if (fs != sampleRate) {
+		fs = sampleRate;
+		init();
```

Comment:
I think we also need to call activate() since it sets some LFO state based on fs. 

This is only ever going to get called the first time an effect is processed after creation, or when the user changes the sample rate on their soundcard (so the sound is going to drop out anyway).